api/rs/slint/mcu.md

```diff
@@ -80,42 +76,42 @@ in a format that's suitable for the software based renderer we're going to use.
+   If there are no crates, you may have to develop your own drivers.
+ * Drive the event loop by querying peripherals for input, forwarding that input into computational modules of your
    application and instructing Slint to render the user interface.
 
-In Slint, the two primary APIs you need to use to accomplish these tasks are the [`slint::platform::Platform`] trait as well as the [`slint::Window`] struct.
+In Slint, the two primary APIs you need are the [`slint::platform::Platform`] trait and the [`slint::Window`] struct.
 In the following sections we're going to cover how to use them and how they integrate into your event loop.
 
 ### The `Platform` Trait
 
 The [`slint::platform::Platform`] trait defines the interface between Slint and platform APIs typically provided by operating and windowing systems.
 
-You need to provide a minimal implementation of this trait and call [`slint::platform::set_platform`] before constructing your Slint application.
+You need to write a minimal implementation of this trait and call [`slint::platform::set_platform`] before constructing your Slint application.
 
 This minimal implementation needs to cover two functions:
 
- * `fn create_window_adapter(&self) -> Result<Rc<dyn WindowAdapter + 'static>, PlatformError>;`: Implement this function to return an implementation of the `WindowAdapter`
-   trait that will be associated with the Slint components you create. We provide a convenience struct [`slint::platform::software_renderer::MinimalSoftwareWindow`]
+ * `fn create_window_adapter(&self) -> Result<Rc<dyn WindowAdapter + 'static>, PlatformError>;`: This function returns an implementation of the `WindowAdapter`
+   trait that will be associated with the Slint components you create. Slint has a convenience struct [`slint::platform::software_renderer::MinimalSoftwareWindow`]
    that implements this trait.
  * `fn duration_since_start(&self) -> Duration`: For animations in `.slint` design files to change properties correctly, Slint needs to know
-   how much time has elapsed between two rendered frames. In a bare metal environment you need to provide a source of time. Often the HAL crate of your
-   device provides a system timer API for this, which you can query in your impementation.
+   how much time has elapsed between two rendered frames. In a bare metal environment you need to have a source of time. Often the HAL crate of your
+   device includes a system timer API, which you can query in your implementation.
 
-There are additional functions in the trait that you can implement, for example to handle debug output, to delegate the event loop or to implement
-the interface to safely deliver events in multi-threaded environments.
+You may override more functions of this trait, for example to handle debug output, to delegate the event loop,
+or to deliver events in multi-threaded environments.
```

Comment:
"Overriding" is a good choice here, indeed better than before and consistent with the Rust language docs. 👍